samples/plugins/db_impersonation/connectionProperties.js

```diff
@@ -0,0 +1,24 @@
+(function propertiesbuilder(attr) {
+    
+    function isEmpty(str) {
+        return (!str || 0 === str.length); 
+    }
+
+    var props = {};
+    props["UID"] = attr[connectionHelper.attributeUsername];
+    props["PWD"] = attr[connectionHelper.attributePassword];
+   
+    if (attr[connectionHelper.attributeTableauServerAuthMode] == connectionHelper.valueAuthModeDBImpersonate) {
+        var str = attr[connectionHelper.attributeTableauServerUser];
+        
+        if (!isEmpty(str)){
+            props["DelegationUID"] = str;
```

Comment:
Add a comment to the doc linked in gh-pages as well: https://impala.apache.org/docs/build/html/topics/impala_delegation.html